src/sbml/Constraint.cpp

```diff
@@ -427,7 +427,7 @@ void
 Constraint::replaceSIDWithFunction(const std::string& id, const ASTNode* function)
 {
   if (isSetMath()) {
-    if (mMath->getType() == AST_NAME && mMath->getId() == id) {
+    if (mMath->getType() == AST_NAME && mMath->getName() == id) {
```

Comment:
it seems like here you have teh same fix included, that are also in the pull request for the comp fix. Are there any differences here? which one should we use?

Reply:
Bah, sorry--this fix should indeed be part of the other pull request.  Did it get in both?  In the end, this line needs to be 'getName', however that works out.